ui/app/services/path-help.js

```diff
@@ -99,64 +109,15 @@ export default Service.extend({
       });
   },
 
-  reducePathsByPathName(pathInfo, currentPath) {
-    const pathName = currentPath[0];
-    const pathDetails = currentPath[1];
-    const displayAttrs = pathDetails['x-vault-displayAttrs'];
-
-    if (!displayAttrs) {
-      return pathInfo;
-    }
-
-    let itemType, itemName;
-    if (displayAttrs.itemType) {
-      itemType = displayAttrs.itemType;
-      let items = itemType.split(':');
-      itemName = items[items.length - 1];
-      items = items.map((item) => dasherize(singularize(item.toLowerCase())));
-      itemType = items.join('~*');
-    }
-
-    if (itemType && !pathInfo.itemTypes.includes(itemType)) {
-      pathInfo.itemTypes.push(itemType);
-    }
-
-    const operations = [];
-    if (pathDetails.get) {
-      operations.push('get');
-    }
-    if (pathDetails.post) {
-      operations.push('post');
-    }
-    if (pathDetails.delete) {
-      operations.push('delete');
-    }
-    if (pathDetails.get && pathDetails.get.parameters && pathDetails.get.parameters[0].name === 'list') {
-      operations.push('list');
-    }
-
-    pathInfo.paths.push({
-      path: pathName,
-      itemType: itemType || displayAttrs.itemType,
-      itemName: itemName || pathInfo.itemType || displayAttrs.itemType,
-      operations,
-      action: displayAttrs.action,
-      navigation: displayAttrs.navigation === true,
-      param: pathName.includes('{') ? pathName.split('{')[1].split('}')[0] : false,
-    });
-
-    return pathInfo;
-  },
-
-  filterPathsByItemType(pathInfo, itemType) {
-    if (!itemType) {
-      return pathInfo.paths;
-    }
-    return pathInfo.paths.filter((path) => {
-      return itemType === path.itemType;
-    });
-  },
-
+  /**
+   * getPaths is used to fetch all the openAPI paths available for an auth method,
```

Comment:
Thank you for adding this documentation!

So this limited to auth methods?

Reply:
Yes, as far as I can tell we only use this for auth backends. But there's nothing stopping us from using it in secret engines, I think